weave-js/src/components/Panel2/PanelTraceTree/PanelTrace.tsx

```diff
@@ -67,11 +67,11 @@ const PanelTraceRender: React.FC<PanelTraceTreeTraceProps> = props => {
         ({row, index}) =>
           opDict({
             name: opPick({obj: row, key: constString('name')}),
-            start_time_ms: opPick({
+            start_time_s: opPick({
```

Comment:
We need to be careful here. `TraceTreeSpanViewer` accepts ms which is also used by the entire current Trace product which uses ms. This layer might need to do a `* 1000`

Reply:
yeah, can fix later though, no one is using PanelTrace yet